cjworkbench/i18n/trans.py

```diff
@@ -174,25 +199,25 @@ def _replace_tags(self, target_message, tags):
 
     def _format_message(self, message, parameters={}, do_escape=True):
         """Substitute parameters into ICU-style message.
-        At this point, ICU is not actually supported (i.e. you can have no plurals, secects, etc).
-        Only variable substitution is supported.
+        You can have variable substitution, plurals, selects and nested messages.
         
         The parameters must be a dict
         """
         if not message:
             return message
+        message_format = MessageFormat(
+            UnicodeString(message), Locale.createFromName(self.locale)
```

Comment:
I think now's the right time to change the `__init__()` function:

* Rename `locale` arg to `locale_id` and `self.locale` to `self.locale_id` should be `locale_id` (it's an ID, right?)
* `self.icu_locale = Locale.createFromName(locale_id)`